code/game/objects/effects/decals/misc.dm

```diff
@@ -121,4 +239,119 @@
 	mouse_opacity = 0
 
 /obj/effect/decal/riverdecal/edge
-	icon_state = "riverdecaledge"
+	icon_state = "riverdecal_edge"
+
+/obj/effect/decal/corsat/symbol
+	icon = 'icons/effects/warning_stripes.dmi'
+	color = "#9c7f42"
+
+/obj/effect/decal/corsat/symbol/omega
+	icon_state = "omega"
+	color = "#534563"
+
+/obj/effect/decal/corsat/symbol/theta
+	icon_state = "theta"
+	color = "#549c42"
+
+/obj/effect/decal/corsat/symbol/gamma
+	icon_state = "gamma"
+	color = "#5d6384"
+
+/obj/effect/decal/corsat/symbol/sigma
+	icon_state = "sigma"
+
+
+/obj/effect/decal/grassdecal
+	name = "grass"
+	gender = PLURAL
+	density = FALSE
+	anchored = TRUE
+	icon = 'icons/turf/ground_map.dmi'
+	icon_state = "grassdecal_edge"
```

Comment:
gender???

Reply:
I don't know, some of the decals also have gender defined and I copypasted, should I also remove gender from the other decals?